src/main/java/frc/robot/operator_interface/FullOperatorConsoleOI.java

```diff
@@ -81,14 +81,19 @@ public double getRotate() {
 
   @Override
   public Trigger getXStanceButton() {
-    return rotateJoystickButtons[4];
+    return rotateJoystickButtons[6];
```

Comment:
This part is on the rotate joystick, which does not contain the x-stance on this year's feature sheet. I just changed the number so that the placement 4 was there for manual intake